CHANGELOG.md

```diff
@@ -56,7 +56,11 @@ To learn more about active deprecations, we recommend checking [GitHub Discussio
 
 Here is an overview of all new **experimental** features:
 
+- **General**: Introduce new Solr Scaler ([#4234](https://github.com/kedacore/keda/issues/4234))
 - **General**: TODO ([#XXX](https://github.com/kedacore/keda/issues/XXX))
+- **CPU/Memory scaler**: Add support for scale to zero if there are multiple triggers([#4269](https://github.com/kedacore/keda/issues/4269))
+- **Prometheus Metrics**: Introduce number of paused ScaledObjects in Prometheus metrics ([#4430](https://github.com/kedacore/keda/issues/4430))
+- **Redis Scalers**: Allow scaling using redis stream length ([#4277](https://github.com/kedacore/keda/issues/4277))
```

Comment:
I think that it should be an error here, there are 3 extra lines added from previous release, aren't?

Reply:
Oh so what line should I remove?

Reply:
I think that only this line is yours: `- **Prometheus Metrics**: Introduce number of paused ScaledObjects in Prometheus metrics ([#4430](https://github.com/kedacore/keda/issues/4430))`, so keep it only

Reply:
Alright I will do that